### PR TITLE
[WIP] Release blocking jobs require TRT approval

### PR DESCRIPTION
**Pull request by brenton**
This PR contains a README with a statement around ownership as well as a
minor refactoring of the TRT alias.  As we ramp up new team members it's
important to have a distinction between reviewers and owners.

### CODE REVIEW

**Comment by brenton** [comment]
@deads2k, Are you OK with the refactoring of the TRT owners alias here?  I removed you from the alias but explicitly left you as an owner in the release gating directory.  Just about everywhere else you are explicitly called out too so I thought it was the right thing to do.

File: core-services/release-controller/_releases/README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Release Gating Jobs.
+
+This directory contains the configuration for [release gating and informing jobs](https://docs.ci.openshift.org/docs/architecture/release-gating/).  Any jobs marked gating must obtain approval from the [Technical Release Team](/OWNERS_ALIASES).
```
Comment (wking) [inline core-services/release-controller/_releases/README.md:3]: to make the semantics of "marked gating" more clear, maybe:

> Any jobs without `optional: true` must obtain...

?

Comment (brenton) [reply core-services/release-controller/_releases/README.md:3]: Let me reword this...

Comment (brenton) [reply core-services/release-controller/_releases/README.md]: Check out the update.

**Comment by bradmwilliams** [comment]
The `release-controller-config` test will pass once: https://github.com/openshift/release/pull/23792 merges

**Comment by bradmwilliams** [comment]
/test build02-dry
They have made this test optional now, so it should pass as well

**Pull request merged**